llm_grok.py

```diff
@@ -60,25 +69,21 @@ class Options(llm.Options):
             le=1,
             default=0.0,
         )
-        max_tokens: Optional[int] = Field(
-            description="The maximum number of tokens to generate in the completion.",
+        max_completion_tokens: Optional[int] = Field(
+            description="The maximum number of tokens to generate, including visible output tokens and reasoning tokens.",
             ge=0,
             default=None,
         )
+
     def __init__(self, model_id):
         self.model_id = model_id
 
     def build_messages(self, prompt, conversation):
         messages = []
-        
+
         if prompt.system:
             messages.append({"role": "system", "content": prompt.system})
-        else:
-            messages.append({
-                "role": "system", 
-                "content": "You are Grok, a chatbot inspired by the Hitchhikers Guide to the Galaxy."
```

Comment:
Remove the default system prompt since it's kind of surprising for users, and changes responses a fair bit